ocrd/ocrd/processor/helpers.py

```diff
@@ -216,6 +340,12 @@ def wrap(s):
                                   (with --page-id, remove only those)
   -p, --parameter JSON-PATH       Parameters, either verbatim JSON string
                                   or JSON file path
+  --server-ip IP                  Host name/IP to listen at. When this value is set,
+                                  --server-port must be set as well.
+  --server-port NUMBER            TCP port to listen at. When this value is set,
+                                  --server-ip must be set as well.
+  --mongo-url URL                 Connection string to a Mongo database. If the processor runs as a server,
+                                  this value must be set.
```

Comment:
That particular place in the help string (between parameter and param-override) is awkward. I suggest moving before the other non-processing options (show-resource, list-resources, dump-json, help, version).

Reply:
Agree.